cin_validator/rules/cin_2023_24/__innit__.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+
+__all__ = [
+    p.stem for p in Path(__file__).parent.glob("*.py") if p.stem != "__init__"
+].append(
+    [
+        p.stem
+        for p in Path("cin_validator/rules/cin2022_23").parent.glob("*.py")
+        if p.stem
+        in [
+            "__init__",
+            "rule_4009",
+            "rule_8525",
+            "rule_8555",
+            "rule_8569",
+            "rule_8863",
+            "rule_8873",
+            "rule_8897",
+        ]
```

Comment:
Hi Will, 
I see where you're going.
No worries. We don't need to specify the file names. If you use the exact init file that is found in the cin2022_23 ruleset folder, this will run too. 
That code ensures that all the python files, found at the same level as the init file, are run. 
Let's start there first.
Commit in the mvp innit file so that you can test the rule changes you're making. Then we can later work on tweaking the innit such that it inherits unchanged rules from the cin2022_23 ruleset.